kedro/runner/sequential_runner.py

```diff
@@ -20,27 +20,17 @@ class SequentialRunner(AbstractRunner):
     """
 
     def __init__(self, is_async: bool = False):
-        """Instantiates the runner classs.
+        """Instantiates the runner class.
 
         Args:
             is_async: If True, the node inputs and outputs are loaded and saved
                 asynchronously with threads. Defaults to False.
 
         """
-        super().__init__(is_async=is_async)
-
-    def create_default_dataset(self, ds_name: str) -> AbstractDataset:
-        """Factory method for creating the default data set for the runner.
-
-        Args:
-            ds_name: Name of the missing data set
-
-        Returns:
-            An instance of an implementation of AbstractDataset to be used
-            for all unregistered data sets.
-
-        """
-        return MemoryDataset()
+        default_dataset_pattern = {"{default}": {"type": "MemoryDataset"}}
```

Comment:
How can I change this if I inherit from `SequentialRunner`?

Reply:
I see what you mean. So I guess you'd prefer it if the `__init__` of all runners take the `extra_dataset_patterns` which you can then overwrite?

Reply:
Yup, that would be nice.

Reply:
Done!